graphql/e2e/directives/docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
       --trace "ratio=1.0;"
 
   lambda:
-    image: dgraph/dgraph-lambda:latest
+    image: namanj1811/lambda-test:v2
```

Comment:
TODO: I will update this once https://github.com/dgraph-io/dgraph-lambda/pull/22 is merged.